app/firebase.server.ts

```diff
@@ -61,4 +61,4 @@ function getFirebaseClientConfig() {
   };
 }
 
-export { app, auth, /* db, */ getFirebaseClientConfig };
+export { app, auth, db, getFirebaseClientConfig };
```

Comment:
This file now exports a `db` object, which can only be used on the server with admin credentials (or faked by the Firebase emulators.)